app/controllers/JourneyCompletionController.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import models.TaxRegime.EpayeRegime
+import play.api.mvc.{ Action, AnyContent, MessagesControllerComponents }
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import util.Logging
+
+import javax.inject.Inject
+
+class JourneyCompletionController @Inject() (cc: MessagesControllerComponents) extends FrontendController(cc) with Logging {
+
+  // just return to the test page for now.
```

Comment:
I would rather create a placeholder pages with dummy content and implement that redirecting to the right pages.
Feel free to ignore it and develop it another time.